website/templates/include/profile/settings_navpanel.mako

```diff
@@ -13,5 +13,10 @@
 
       <li class="${'active' if current_page == 'notifications' else ''}">
         <a href="${ '#' if current_page == 'notifications' else web_url_for('user_notifications') }">Notifications</a></li>
+
+      % if dev_mode:
```

Comment:
I'm not sure this should be `dev_mode`. When `dev_mode = True`, it means that the application is in a non-production environment. 

I would think we'd have a separate settings to enable/disable developer apps.